SPECS/Cython/Cython.spec

```diff
@@ -1,53 +1,85 @@
 %{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
 Summary:       C extensions for Python
 Name:          Cython
 Version:       0.28.5
-Release:       7%{?dist}
+Release:       8%{?dist}
 Group:         Development/Libraries
 License:       ASL 2.0
 URL:           https://cython.org
 #Source0:      https://github.com/cython/cython/archive/%{version}.tar.gz
 Source0:       %{name}-%{version}.tar.gz
 Patch0:        fix_abc_tests.patch
+Patch1:        cython_testfix_with_outer_raising.patch
 Vendor:        Microsoft Corporation
 Distribution:  Mariner
 BuildRequires: python2-devel
 BuildRequires: python2-libs
 BuildRequires: python-xml
 Requires:      python2
+Provides:      python2-%{name}
+Provides:      cython
 
 %description
 Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (based on Pyrex). It makes writing C extensions for Python as easy as Python itself.
 
+%package -n python3-%{name}
+Summary:       C extensions for Python
```

Comment:
Maybe explicitly say "Python 3" here and "Python 2" in the default package's `Summary` tag.